### ISSUE TITLE
`'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

### ISSUE THREAD

**Issue by chenxsan** [mdx-js/eslint-mdx#331]
### Subject of the issue

Using `'` in javascript code block inside mdx shows an error as the title:

<pre>
&lt;div>
```js
import foo from './foo.js'
```
&lt;/div>
</pre>

<img width="913" alt="" src="https://user-images.githubusercontent.com/1091472/128655411-18c0286e-6d03-4cdd-9118-fa09ffb31553.png">

### Your environment

*   **OS**: 
*   **Packages**: 
*   **Env**: 

### Steps to reproduce

1. Clone code from https://github.com/webpack/webpack.js.org/pull/5233
2. Run `yarn` to install
3. open `src/content/index.mdx`, and append code below:

<pre>
&lt;div>
```js
import foo from './foo.js'
```
&lt;/div>
</pre>

### Expected behaviour

No lint error.

### Actual behaviour

A lint error.

**Comment by JounQin** [comment]
See https://github.com/mdx-js/eslint-mdx/issues/207#issuecomment-782716731

Short answer, it can not be fixed in `eslint-plugin-mdx@v1`, and I don't have time to work on v2 temporarily.

Workaround:

````mdx
<!-- eslint-disable react/no-unescaped-entities -->

<div>
```js
import foo from './foo.js'
```
</div>

<!-- eslint-enable react/no-unescaped-entities -->
````

**Comment by JounQin** [comment]
Or:

````mdx
<!-- example.mdx -->
```js
import foo from './foo.js';
```
````

````mdx
<!-- index.mdx -->
import Example from './example.mdx'

<div>
  <Example />
</div>
````

**Comment by chenxsan** [comment]
> Or:
> 
> ```gfm
> 
> ```js
> import foo from './foo.js';
> ```
> ```
> 
> ```gfm
> 
> import Example from './example.mdx'
> 
> <div>
>   <Example />
> </div>
> ```

Yeah, this is how I work around it at the moment.
